skins/cat17/faq-page/src/Faq.vue

```diff
@@ -71,167 +71,25 @@ export default {
-							'nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in ' +
-							'reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. '
-						},
-						{
-							question: 'Wikimedia-Organisation Question 2',
-							answer: 'Wikimedia-Organisation Answer 2 ' +
-							'Lorem ipsum dolor sit amet, consectetur adipiscing elit, ' +
-							'sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. ' +
-							'Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris ' +
-							'nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in ' +
-							'reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. '
-						}
-					]
-				},
-				{
-					name: 'Wikimedia-Projekte',
-					content: [
-						{
-							question: 'Wikimedia-Projekte Question',
-							answer: 'Wikimedia-Projekte Answer ' +
-							'Lorem ipsum dolor sit amet, consectetur adipiscing elit, ' +
-							'sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. ' +
-							'Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris ' +
-							'nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in ' +
-							'reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. '
-						},
-						{
-							question: 'Wikimedia-Projekte Question 2',
-							answer: 'Wikimedia-Projekte Answer 2 ' +
-							'Lorem ipsum dolor sit amet, consectetur adipiscing elit, ' +
-							'sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. ' +
-							'Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris ' +
-							'nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in ' +
-							'reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. '
-						}
-					]
-				},
-				{
-					name: 'Spendenverwendung',
-					content: [
-						{
-							question: 'Spendenverwendung Question',
-							answer: 'Spendenverwendung Answer ' +
-							'Lorem ipsum dolor sit amet, consectetur adipiscing elit, ' +
-							'sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. ' +
-							'Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris ' +
-							'nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in ' +
-							'reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. '
-						},
-						{
-							question: 'Spendenverwendung Question 2',
-							answer: 'Spendenverwendung Answer 2 ' +
-							'Lorem ipsum dolor sit amet, consectetur adipiscing elit, ' +
-							'sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. ' +
-							'Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris ' +
-							'nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in ' +
-							'reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. '
-						}
-					]
-				}
-			]
+			topicTitle: '',
+			content: faqContent,
+			messages: faqMessages,
```

Comment:
I think you can remove the two previous lines. `content` and `messages` are already properties, there is no need to put them in the state, right? Also, where do those variables come from?

Reply:
> content and messages are already properties, there is no need to put them in the state, right?

true
> Also, where do those variables come from?

They come from `faq.html.twig`